doc/manual/manual.md

```diff
@@ -2505,13 +2505,15 @@ m <- llvm_load_module "./test.bc";
 
```

Comment:
The comment above (which GitHub won't let me highlight) says:

```xml
<!-- This should (partially) match intTests/test0036_globals/test.saw -->
```

But there is no file with that name, AFAICT. The closest thing that I've found is [this test case](https://github.com/GaloisInc/saw-script/blob/4a695fba856ec8a29563d8a8ba7a01c47569b943/intTests/test0036_global/test.c), but that tests unsigned integers rather than signed ones. Should we create a dedicated example/test case for this and update the comment to point to it? (Similarly for the other two "This should (partially) match" comments above this one.)

Reply:
Good catch. I added tests for these.